src/Azure/Orleans.Streaming.EventHubs/Providers/Streams/EventHub/EventHubConstants.cs

```diff
@@ -2,7 +2,6 @@ namespace Orleans.ServiceBus.Providers
 {
     internal class EventHubConstants
     {
-        public readonly static string StartOfStream =
-            Microsoft.Azure.EventHubs.PartitionReceiver.StartOfStream;
```

Comment:
why this change? there isn't something like `Microsoft.Azure.EventHubs.PartitionReceiver.StartOfStream` in new pkg anymore?

Reply:
No, they don't use '-1' as a signal offset for the start of the partition, but we still need it to not break backwards compatibility with checkpoint entities that may have been written with it.